project/JavalibIRCleaner.scala

```diff
@@ -39,6 +39,9 @@ object JavalibIRCleaner {
   private val writeReplaceMethodName =
     MethodName("writeReplace", Nil, ClassRef(ObjectClass))
 
+  private lazy val baseDirectoryURI =
+    new java.io.File(".").getAbsoluteFile().toURI()
```

Comment:
Could we inject this? (And take the root directory of the root project?)

Reply:
Done.